docs/styles.less

```diff
@@ -1598,4 +1598,15 @@ Spark table documentation styles
     max-width: 350px;
     border-radius: 0;
     z-index: 1;
+}
+
+.ux-icon-component-example ux-icon {
```

Comment:
We only put global styles in this stylesheet. These styles only affect a single documentation section so should go in the component stylesheet.

Also any styles you use in the documentation will also need to be added to the plunker.

Also the `nth-of-type(3)` selector is very rigid, if we ever wanted to add additional icons to the example the styling could then be wrong. A better way would be to have a container div element that groups the icons and adds `margin-right`.